frontend/apps/marketing/src/components/contentful/card/Card.tsx

```diff
@@ -79,11 +79,11 @@ const Card: React.FC<CardProps> = ({
         {primaryButton && (
           <Button
             {...primaryButton}
-            text={primaryButton.fields.label}
-            href={primaryButton.fields.primaryTarget}
-            ariaLabel={primaryButton.fields.ariaLabel}
+            text={primaryButton?.fields?.label}
```

Comment:
The first optional chain `primaryButton?` isn't needed because we already defend against falsy values on 79. The subsequent optional chain is fine though, but an optimization could be to change 79 to `primaryButton.fields? &&` 

The reason is that each optional chain `?` is an equivalent if/else statement. Using it many times is not very performant as it does the if/else multiple times.